dataikuapi/dss/savedmodel.py

```diff
@@ -330,9 +333,10 @@ def set_core_metadata(self,
         #if get_features_from_signature:
         #    raise Exception("Get features from signature is not yet implemented")
 
-        # TODO: Add support for features_list, with validation
+        if features_list is not None:
+            metadata["features"] = features_list
```

Comment:
One common mistake that our beloved users do is passing a list of strings as `features_list`, the strings being the names of the features. And it tooks us some time to spot the error.

So, I would also have a check on the content of the `features_list`, checking that each item in the list is an object with a `name` and a `type` properties.

Reply:
Good call I did it !